model/src/w3gdatmd.F90

```diff
@@ -1566,88 +1501,59 @@ SUBROUTINE W3NMOD ( NUMBER, NDSE, NDST, NAUX )
+  !> @param[in] mbac number of SMC Arctic boundary cells. (SMC switch only.)
+  !> @param[in] mspec Number of discrete spectral bins (not SMC specific). (SMC switch only.)
+#endif
+  !>
+  !> @author H. L. Tolman  @date 10-Dec-2014
+  !>
   SUBROUTINE W3DIMX  ( IMOD, MX, MY, MSEA, NDSE, NDST   &
 #ifdef W3_SMC
        , MCel, MUFc, MVFc, MRLv, MBSMC    &
        , MARC, MBAC, MSPEC                &
 #endif
        )
-    !/
-    !/                  +-----------------------------------+
-    !/                  | WAVEWATCH III           NOAA/NCEP |
-    !/                  |           H. L. Tolman            |
-    !/                  |                        FORTRAN 90 |
-    !/                  | Last update :         10-Dec-2014 |
-    !/                  +-----------------------------------+
-    !/
-    !/    24-Jun-2005 : Origination.                        ( version 3.07 )
-    !/    18-Jul-2006 : Add input grids.                    ( version 3.10 )
-    !/    05-Oct-2006 : Add filter to array pointers.       ( version 3.10 )
-    !/    02-Feb-2007 : Add FLAGST.                         ( version 3.10 )
-    !/    30-Oct-2009 : Implement run-time grid selection.  ( version 3.14 )
-    !/                  (W. E. Rogers & T. J. Campbell, NRL)
-    !/    30-Oct-2009 : Implement curvilinear grid type.    ( version 3.14 )
-    !/                  (W. E. Rogers & T. J. Campbell, NRL)
-    !/    30-Oct-2009 : Implement unstructured grids        ( version 3.14.1)
-    !/    03-Sep-2012 : Clean up of UG grids                ( version 4.08 )
-    !/    10-Dec-2014 : Add checks for allocate status      ( version 5.04 )
-    !/
-    !  1. Purpose :
-    !
-    !     Initialize an individual spatial grid at the proper dimensions.
-    !
-    !  2. Method :
-    !
-    !     Allocate directly into the structure array GRIDS. Note that
-    !     this cannot be done through the pointer alias!
-    !
-    !  3. Parameters :
-    !
-    !     Parameter list
-    !     ----------------------------------------------------------------
-    !       IMOD    Int.   I   Model number to point to.
-    !       NDSE    Int.   I   Error output unit number.
-    !       NDST    Int.   I   Test output unit number.
-    !       MX, MY, MSEA       Like NX, NY, NSEA in data structure.
-    !     ----------------------------------------------------------------
-    !
-    !  4. Subroutines used :
-    !
-    !       See module documentation.
-    !
-    !  5. Called by :
-    !
-    !      Name      Type  Module   Description
-    !     ----------------------------------------------------------------
-    !      W3IOGR    Subr. W3IOGRMD Model definition file IO program.
-    !      WW3_GRID  Prog.   N/A    Model set up program.
-    !     ----------------------------------------------------------------
-    !
-    !  6. Error messages :
-    !
-    !     - Check on input parameters.
-    !     - Check on previous allocation.
```

Comment:
These guys are not retained.

Reply:
section `6. Error messages`

Reply:
I'm not sure that has much value, but I've added it to the subroutine documentation...